### DISCUSSION TITLE
Tool Redesign

### DISCUSSION THREAD

**Discussion by kylecorry31** [kylecorry31/Trail-Sense#1867] [New Feature Ideas]
Now that the Photo Maps feature is almost complete (at least the MVP of it), I'm switching my focus to redesigning existing tools to make them intuitive to use and easy to read at a glance. 

You can find more of my reasoning and goals here: #1792
You can find specific redesign ideas here: #1795 and #1796
Astronomy before and after: #1794 

In the last release (5.0.0), I redesigned the Astronomy tool with this in mind and I feel that it is much easier to read and only shows useful information - though I definitely have bias. 

I also know that anyone using GitHub to answer these questions has at least a slight technical bias - meaning it may be easier for you to figure out how to use a tool or, like me, you may want to see details that the average person does not care about. So if there was a tool you had to experiment a little to get working or had to look something up to understand what it meant, I would love to know.

My questions:
- How do you like the new Astronomy tool redesign?
- Does the Astronomy tool provide all the information you need at a glance while on the trail? If not, what is missing?
- What tools in Trail Sense are not very intuitive?
- What tools in Trail Sense are hard to read at a glance (cluttered)? What information is not necessary to display prominently?

**Comment by michaelblyons** [comment]
### Astronomy redesign

Seems great to me. I'm primarily glad to be rid of the blue bar above the horizon. It was very bright with the dark theme outside at night.

I originally installed your app to help my kids find the moon (and/or how late they would have to stay up to see the moon).

I thought I would miss seeing the order of moonrise and sunset when you dropped the huge list of rows, but the new format is great.

### Tool confusion

#### Packing list 
I keep skipping past the category drop-down because I misinterpret "Other" as a form section, not a selectable option. Then I see the item on the actual list and go back to edit the item and pick the category.

#### Lightning timer 
Is it possible to disable or extend the screen timeout on this tool? I have the app up while I look at the sky waiting for the next flash, and then I tap the screen, only to find that it is dark.

**Reply by kylecorry31** [reply]
Thank you for the feedback!

Here's the new issues to track those changes (I've also added packing list to the tool redesign as I think I can improve this more):
- Packing list: https://github.com/kylecorry31/Trail-Sense/issues/1889
- Lightning: https://github.com/kylecorry31/Trail-Sense/issues/1890

Both are pretty simple changes, so I'll add them in the next release.

**Comment by HidingInEmptySpace** [comment]
### How do you like the new Astronomy tool redesign?
Love it.

### Does the Astronomy tool provide all the information you need at a glance while on the trail? If not, what is missing?

1. The Calibration of the compass could also be done by selecting waypoints that are in sight. For example towers that lure out of the forest could then be used to calibrate the compass, as long as the current position of the user is known. 

2. The Calibration feature should apply the new Calibration to all other tools if wanted by the user. Which is userfull for example when sitting inside a car or inside a cave or boat where the magnetic field is being disturbed.

3. "Use only gyroscope after calibrating" should also be applyable to all tools also due to magnetic disturbance (Maybe reenable using the compass after certain interval ?)

### What tools in Trail Sense are not very intuitive?

All tools are great designed.
The icon of the sunset alarm should be something other that an arrow. Maybe a sun or an alarm clock ?

### What tools in Trail Sense are hard to read at a glance (cluttered)?

None on my opinion. You did a really good job on that !

### What information is not necessary to display prominently?

The red "location disabled" notification bar under the navigation tool always reappears after reopening the navigation tab. Maybe its intentional ?

**Reply by kylecorry31** [reply]
Thank you for the feedback and suggestions!

1. https://github.com/kylecorry31/Trail-Sense/issues/2068 - this will allow you to calibrate using beacons
2. https://github.com/kylecorry31/Trail-Sense/issues/2419 - this will allow calibrating the compass on tools other than Augmented Reality
3. https://github.com/kylecorry31/Trail-Sense/issues/2419 - this will also include the ability to use the gyroscope only after calibrating (until the tool is closed - same as Augmented Reality)
4. https://github.com/kylecorry31/Trail-Sense/issues/2420 - this will update the sunset alert icon
5. You can disable these errors in Settings > Errors. But it indicates that your GPS is not turned on, so be cautious because your location may not be correct.
